src/main/java/com/openshift/jenkins/plugins/pipeline/model/IOpenShiftPlugin.java

```diff
@@ -624,8 +624,7 @@ default boolean annotateJobInfoToResource(IClient client, TaskListener listener,
         return annotated;
     }
 
-    default String httpGet(boolean chatty, TaskListener listener,
-                           Map<String, String> overrides, String urlString) {
+    static String httpGet(boolean chatty, TaskListener listener, Map<String, String> overrides, String urlString, String authToken, String displayName, DefaultClient client, Auth auth, String apiURL) {
```

Comment:
Let's create a separate class called `HttpUtils` in the model package and put this static method there.

Then leave `httpGet` as a default method in `IOpenShiftPlugin`, where in that method you call `HttpUtils.httpGet`